components/01-globals/variables/_variables.scss

```diff
@@ -1,15 +1,15 @@
 // colors
 
-$color-main1: #f26322;
-$color-main2: #ff5000;
-$color-main3: #883008;
+$color-orange: #f26322;
+$color-orange-second: #ff5000;
+$color-orange-dark: #883008;
 $color-black: #000000;
 $color-white: #ffffff;
-$color-accent: #2e7501;
+$color-green: #2e7501;
 $color-aqua: #c0e9fd;
-$color-gray1: #9ea3a4;
-$color-gray2: #666666;
-$color-gray3: #c2c2c2;
+$color-gray: #9ea3a4;
+$color-gray-second: #666666;
+$color-gray-third: #c2c2c2;
```

Comment:
But... it's okay for gray colors. Usually, we have like z 10 shades of gray, so naming them sucks. You might try to replace it with some descriptive color name i.e. `color-graphite`, `color-spacegray`(you may try to find something like this and use this weird names https://s-media-cache-ak0.pinimg.com/736x/dc/c8/bc/dcc8bca2dc120a018e208be8ae72b24f.jpg )